### ISSUE TITLE
the bug of api $("#jqGrid").jqGrid('getRowData');

### ISSUE THREAD

**Issue by quoniammm** [tonytomov/jqGrid#831]
this is my version of jqgrid.js
```
/* 
 * jqGrid  5.1.1  
 * Copyright (c) 2008, Tony Tomov, tony@trirand.com 
 *  License: http://guriddo.net/?page_id=103334 
 *
 * Modules: grid.base.js; jquery.fmatter.js; grid.common.js; grid.formedit.js; grid.filter.js; grid.inlinedit.js; grid.celledit.js; jqModal.js; jqDnR.js; grid.subgrid.js; grid.grouping.js; grid.treegrid.js; grid.pivot.js; grid.import.js; grid.utils.js; grid.jqueryui.js; jquery.sortable.js;
 */
```

when i use `jqGrid('getRowData');` The resulting data is always missing one.

Therefore, i doubt wheather this is a bug about jqgrid.

when i change the source code of jqgrid, the problem is solved. Who can tell me why?Please help me~

```
 if (null == a) f = !0, c = [], d = j.rows.length; // the change
if (null == a) f = !0, c = [], d = j.rows.length - 1; // the source code.
```

**Comment by tonytomov** [comment]
Hello,

Thanks. It is fixed and available in v5.2 : https://github.com/tonytomov/jqGrid/commit/77ce82fc017af0405a41efee503a2ff345fed280

Regards,
Tony

**Comment by quoniammm** [comment]
@tonytomov  Thank you very much, this problem is solved.
